packages/context/test/unit/context.ts

```diff
@@ -253,15 +253,15 @@ describe('Context', () => {
 
   describe('get', () => {
     it('returns a promise when the binding is async', async () => {
-      ctx.bind('foo').to(Promise.resolve('bar'));
+      ctx.bind('foo').toDynamicValue(() => Promise.resolve('bar'));
```

Comment:
This can be further simplified as `ctx.bind('foo').toDynamicValue(async () => 'bar');`

Reply:
Agreed.

I personally prefer `Promise.resolve` in this particular case, because `async` functions incur overhead in terms of the generated code (either explicitly by `tsc` for Node.js 6.x or implicitly by V8).